src/main/java/ca/nines/ise/validator/SectionCoverageValidator.java

```diff
@@ -44,20 +44,20 @@
 
   ValidatorStack<StartNode> nodeStack;
   ArrayList<StartNode> coverAll;
-  ArrayList<StartNode> acts;
-  ArrayList<StartNode> scenes;
-  ArrayList<StartNode> pages;
+  Boolean acts;
+  Boolean scenes;
+  Boolean pages;
```

Comment:
Do these three variables need to be Boolean objects? It looks like they can be primitive booleans.